selfdrive/test/helpers.py

```diff
@@ -47,6 +47,11 @@ def wrap(self, *args, **kwargs):
 
 @contextlib.contextmanager
 def processes_context(processes, init_time=0, ignore_stopped=None):
+  # need to dynamically import this in case there is some configuration we set within a test
+  if "openpilot.selfdrive.manager.process_config" in sys.modules:
+    del sys.modules["openpilot.selfdrive.manager.process_config"]
+  from openpilot.selfdrive.manager.process_config import managed_processes
```

Comment:
this is super hacky. if this is what the intermediate function was for, that's fine